tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -305,7 +313,11 @@ def test_5_inc_snapshots(network_config,
                          bin_cloner_path,
                          bin_vsock_path,
                          test_fc_session_root_path):
-    """Test scenario: 5 incremental snapshots with disk intensive workload."""
+    """
+    Test scenario: 5 incremental snapshots with disk intensive workload.
```

Comment:
Not sure what is the intensive workload here. Does not seem to be different than the full snapshot scenario.